.travis.yml

```diff
@@ -2,5 +2,4 @@ language: clojure
 lein: lein2
 script: lein2 test-all
 jdk:
-  - openjdk7
-  - openjdk6
+  - oraclejdk8
```

Comment:
I've been reading through the documentation of lein-doo. It looks like even Java libraries like Rhino require an external executable for some reason! 

I think I'll be happy to go back to Phantom, if Rhino or other Java libraries are no better supported by doo. I'd like to keep openjdk 7 just so that the Clojure version is tested with an older version of Java, though I don't support it matters.